c/include/libsbp/acquisition.h

```diff
@@ -49,7 +48,7 @@ typedef struct SBP_ATTR_PACKED {
 
 /** Deprecated
  *
-* Deprecated.
+ * Deprecated.
```

Comment:
do we need the second `Deprecated` here?  If not, it is easy to detect it is repeated and remove it, or just remove it from the yaml files.

Reply:
I'm good with removing it